tools/cellprofiler_v4/cellprofiler.xml

```diff
@@ -0,0 +1,244 @@
+<tool id="cp_cellprofiler" name="CellProfiler" version="@CP_VERSION@+galaxy@VERSION_SUFFIX@">
+    <description>run a CellProfiler pipeline</description>
+    <macros>
+        <import>macros.xml</import>
+        <token name="@VERSION_SUFFIX@">0</token>
+        <xml name="test_assert_content" token_n="291">
+            <assert_contents>
+                <has_n_lines n="@N@" />
+            </assert_contents>
+        </xml>
+    </macros>
+    <expand macro="cp_requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+        bash '$script_file' &&
+        cellprofiler -c -r --file-list list.txt -o ./output -p '$pipeline' 
+        #if '$detailed_output'
```

Comment:
```suggestion
        #if $detailed_output
```

Reply:
I tried to run the test, but all of them failed. I got this error:

![Screenshot from 2021-09-22 16-41-52](https://user-images.githubusercontent.com/26678117/134365583-e3a95cd9-6235-4bd6-a7f9-6d4685fa15a7.png)

Reply:
Thanks  @gallardoalba 
I couldn't find where is this /home/laptop directory, to be honest.
I tested it with planemo and the results are green. Could you please point me to the right direction?
Thanks very much

Reply:
but if you look at the output of the CI in github, the actual error is that during CI process, the tests in xml failed because cellprofiler doesn't produce any output files. 

My guess is that this probably caused by the python version in ci.yml, as it is 3.7 in CI but cellprofiler 4 only works with 3.8.

I tested it locally with clean env, both using `planemo t cellprofiler.xml` and `planemo t --docker cellprofiler.xml` all get green test results. so Im out of ideas on this :)


![ubuntu@galaxytools-dev_ ~_galaxytools_tools_cellprofiler_v4 22_09_2021 17_39_17](https://user-images.githubusercontent.com/1610192/134375777-a1ce404e-013c-4157-89e4-525909bfefe3.png)

Reply:
now I saw the same /home/runer/.cache permission error in Github CI output...
hmm...